CONTRIBUTING.md

```diff
@@ -0,0 +1,8 @@
+# S Contributing Guidelines
+
+## Rules
+
+There are a few basic ground-rules for contributors to `s`:
+
+1. When adding a `provider` use for each a **new single** commit.
+2. Use gofmt on all files.
```

Comment:
Also would say that for refactor commits to base code that it should be discussed prior to PR.